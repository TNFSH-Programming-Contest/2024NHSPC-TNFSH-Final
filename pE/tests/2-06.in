1 7996402 5696786 2272805
7009129 947938 947938
24
