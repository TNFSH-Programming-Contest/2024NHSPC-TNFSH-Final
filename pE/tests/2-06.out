3217647190031100
